exercises/resistor-significants/description.md

```diff
@@ -0,0 +1,18 @@
+Resistors have color coded bands, where each color maps to a number. The first 2 bands of a resistor have a simple encoding scheme: each color maps to a single number.
+
+These colors are encoded as follows:
+
+- Black: 0
+- Brown: 1
+- Red: 2
+- Orange: 3
+- Yellow: 4
+- Green: 5
+- Blue: 6
+- Violet: 7
+- Grey: 8
+- White: 9
+
+Combining the numbers of the first two bands gives one its significant value. As an example, if the first band is blue (6) and the second band is grey (8), the significant value is 68. 
+
```

Comment:
```suggestion
Combining the numbers of the first two bands gives one the significant figures of the value in Ohm.
```
And in the example: the value is `68`

Reply:
Or: the value to be returned is `68` . (To avoid the problem of what we should call this part. I'm afraid significant value may be misleading or confusing for its meaning in statistics. )